src/devices/OneWire/OneWireBus.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Iot.Device.OneWire
+{
+    /// <summary>
+    /// Represents a 1-wire bus.
+    /// </summary>
+    public partial class OneWireBus
+    {
+        internal OneWireBus(string deviceId)
+        {
+            DeviceId = deviceId;
+        }
+
+        /// <summary>
+        /// The 1-wire device id.
```

Comment:
should this be bus id?

Reply:
Sure, will change, thought of bus as a device too like in Linux drivers. But agree bus id probably better here.